src/components/PersonRecord/index.js

```diff
@@ -44,7 +48,22 @@ const PersonRecord = ({
       {`Age: ${age}`}
     </Box>
 
-    <Box> 
+    <Box tag="span" pad={{horizontal: 'medium'}}>
+      {`Gender: ${gender}`}
+    </Box>
+
+    <Box tag="span" pad={{horizontal: 'medium'}}>
+      {`Ethnicity: ${ethnicity}`}
+    </Box>
+    <Box tag="span" pad={{horizontal: 'medium'}}>
+      {`Currently Employed: ${employmentStatus}`}
+    </Box>
+
+    <Box tag="span" pad={{horizontal: 'medium'}}>
+      {`Children: ${(familyMembersChildren)}`}
```

Comment:
I think this may break. With backticks, you need to format like this `${familyMembersChildren}` like you did to the others. It may not like the parentheses. If you want to wrap the result in parentheses, do this:
`{`Children: (${familyMembersChildren})`}`